src/pipelines.py

```diff
@@ -79,14 +81,21 @@ def postprocessing(model, label_encoder, config):
                          input_steps=[model],
                          experiment_directory=config.env.cache_dirpath)
 
-    postprocessor = Step(name='postprocessor',
-                         transformer=IdentityOperation(),
-                         input_steps=[model, label_decoder],
-                         adapter=Adapter({'predictions': E('model', 'predictions'),
-                                          'label_mapping': E('decoder', 'label_mapping'),
-                                          }),
-                         experiment_directory=config.env.cache_dirpath)
-    return postprocessor
+    decoder = Step(name='decoder',
+                   transformer=DataDecoder(),
```

Comment:
@apyskir the decoder I wrote needed labale_encoder.transformer as input. How does this one work?